core/src/test/scala/integration/kafka/api/AuthorizerIntegrationTest.scala

```diff
@@ -108,6 +107,8 @@ class AuthorizerIntegrationTest extends BaseRequestTest {
   val producerConfig = new Properties
   val numRecords = 1
 
+  val adminClients = Buffer[AdminClient]()
```

Comment:
Isn't it tearDown() called after each @Test (at least this is true in java unit tests). I think we just need one adminClient that we close in `tearDown`.

Reply:
Not every test needs AdminClient.

Reply:
Hmm, could we do something like the following?

```
class AuthorizerIntegrationTest {
   val adminClient: AdminClient = null

   @After
   def tearDown(): Unit = {
      if (adminClient != null)
          adminClient.close()
   }
}
```